File: RealEstateTransaction.API/RealEstateTransaction.Models/Transaction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RealEstateTransaction.Models
{
    public class Transaction
    {
        public int id { get; set; }
        public string street { get; set; }
        public string city { get; set; }
        public string zip { get; set; }
        public string state { get; set; }
        public int beds { get; set; }
        public int baths { get; set; }
        public string sq__ft { get; set; }
        public string type { get; set; }
        public string sale_date { get; set; }
        public decimal price { get; set; }
        public decimal latitude { get; set; }
        public decimal longitude { get; set; }
    }
}

File: RealEstateTransactionApp/src/app/model/transaction.ts
export interface Transaction {
  id: number;
  street: string;
  city: string;
  zip: string;
  state: string;
  beds: number;
  baths: number;
  sq__ft: string;
  type: string;
  sale_date: string;
  price: number;
  latitude: number;
  longitude: number;
  isEdit: boolean;
}

export const TransactionColumns = [
  {
    key: 'street',
    type: 'text',
    label: 'Street',
    required: true,
  },
  {
    key: 'city',
    type: 'text',
    label: 'City',
  },
  {
    key: 'zip',
    type: 'text',
    label: 'Zip',
    required: true,
  },
  {
    key: 'state',
    type: 'text',
    label: 'State',
  },
  {
    key: 'beds',
    type: 'number',
    label: 'Beds',
  },
  {
    key: 'baths',
    type: 'number',
    label: 'Baths',
  },
  {
    key: 'sq__ft',
    type: 'text',
    label: 'SQ_FT',
  },
  {
    key: 'type',
    type: 'text',
    label: 'Type',
  },
  {
    key: 'sale_date',
    type: 'date',
    label: 'Sale Date',
  },
  {
    key: 'price',
    type: 'number',
    label: 'Price',
  },
  {
    key: 'latitude',
    type: 'number',
    label: 'Latitude',
  },
  {
    key: 'longitude',
    type: 'number',
    label: 'Longitude',
  },
  {
    key: 'isEdit',
    type: 'isEdit',
    label: '',
  },
];
